peps/pep-0752.rst

```diff
@@ -403,23 +403,45 @@ Footnotes
 
 .. [2] Some examples of projects that have many packages with a common prefix:
 
-   - `MkDocs <https://github.com/mkdocs/mkdocs>`__ is a documentation framework
-     based on Markdown files. They have the concept of
-     `plugins <https://www.mkdocs.org/dev-guide/plugins/>`__ which may be
-     developed by anyone and by convention are prefixed by ``mkdocs-``.
+   - `Django <https://www.djangoproject.com>`__ is one of the most widely used
+     frameworks in existence. They have the concept of `middleware`__ which
+     allows for third-party packages to modify the request/response cycle.
+     These packages are by convention prefixed by ``django-``.
```

Comment:
@ofek @warsaw Django apps are not limited to middlewares only, even though it's one of the ways to extend Django-based projects it seems odd to call it out here.

Reply:
Thanks! What did you think I should say here?

Reply:
Something like this (forgive the rST mangling)?

```suggestion
   - `Django <https://www.djangoproject.com>`__ is one of the most widely used
     frameworks in existence and has a conecpt of "`reusable apps 
     <https://docs.djangoproject.com/en/5.1/intro/reusable-apps/>`__",
     which are commonly installed via `third-party packages 
     <https://djangopackages.org/>`__ that implement a subset of functionality
     to extend Django-based website projects. These packages are by convention 
     prefixed by ``django-`` or ``dj-``.
```

Reply:
https://github.com/python/peps/pull/3916